plugins/inputs/systemd_units/README.md

```diff
@@ -198,15 +199,15 @@ were removed, tables are hex aligned to keep some space for future values
 ### Output in non-detailed mode
 
 ```text
-systemd_units,host=host1.example.com,name=dbus.service,load=loaded,active=active,sub=running load_code=0i,active_code=0i,sub_code=0i 1533730725000000000
-systemd_units,host=host1.example.com,name=networking.service,load=loaded,active=failed,sub=failed load_code=0i,active_code=3i,sub_code=12i 1533730725000000000
-systemd_units,host=host1.example.com,name=ssh.service,load=loaded,active=active,sub=running load_code=0i,active_code=0i,sub_code=0i 1533730725000000000
+systemd_units,host=host1.example.com,name=dbus.service,load=loaded,active=active,sub=running,user=telegraf load_code=0i,active_code=0i,sub_code=0i 1533730725000000000
```

Comment:
Given that `user` tag might not be present in the output for all times, should it be listed in this section?

Reply:
You might provide a second output example stating that this is for the user-scoped output. Would that work?